src/client/LocalBlockReader.cpp

```diff
@@ -85,11 +85,14 @@ LocalBlockReader::LocalBlockReader(const shared_ptr<ReadShortCircuitInfo>& info,
         case ChecksumTypeProto::CHECKSUM_CRC32:
         case ChecksumTypeProto::CHECKSUM_CRC32C:
 #if defined(__SSE4_2__) && defined(__LP64__)
+#if !((defined(__PPC64__) || defined(__powerpc64__)) && (__BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__))
```

Comment:
it is a little strange that below under this condition you put
```
 if (HWCrc32c::available()) {
                 checksum = shared_ptr<Checksum>(new HWCrc32c());
```
but here this condition turns out to be in `else` part. Is it a mistake?

Reply:
yes, I have corrected it.

Reply:
also - why it is needed? There is no `HWCrc32c::available()` under powerPC ? But if you made such mistake - it means it built fine with this mistake as I assume I should have tested it locally.

Reply:
yes, there is no hardware CRC32 instruction not available under powerPC.
It has been tested locally. While delivering the changes from local branch to delivery branch it has been missed.